python_modules/libraries/dagster-components/dagster_components/components/dbt_project/component.py

```diff
@@ -68,25 +74,37 @@ def resolve_schema_to_post_processors(
 
 
 def resolve_op_spec(context: ResolutionContext, schema: DbtProjectSchema) -> Optional[OpSpec]:
-    return resolve_as(schema.op, target_type=OpSpec, context=context) if schema.op else None
+    return (
+        resolve_schema_to_resolvable(
```

Comment:
this will need a better name at some point

Reply:
for sure. I want to a naming pass once we see the system in its totality